### DISCUSSION TITLE
请问你们怎么通过mysqlcdc2.3版本把表字段描述（COMMENT）获取到

### DISCUSSION THREAD

**Discussion by Jetol** [apache/flink-cdc#1991] [Q&A]
请问你们把mysql全量的数据同步到另一个postgresql库，怎么通过mysqlcdc获取到字段描述（COMMENT）？我创建JsonDebeziumDeserializationSchema增加参数true，但是获取的json个对象没有字段描述信息
![image](https://user-images.githubusercontent.com/20227903/224630088-903e3bc4-2bad-4e08-81c6-6107ad49c448.png)

**Comment by cYfaris** [comment]
不可以吧 还有这种操作？
